packages/dds/tree/src/test/utils.ts

```diff
@@ -416,7 +410,7 @@ export class TestTreeProviderLite {
 			flushMode,
 		});
 		assert(trees >= 1, "Must initialize provider with at least one tree");
-		const t: SharedTreeWithConnectionStateSetter[] = [];
+		const t: ISharedTree[] = [];
```

Comment:
Why the shift from SharedTreeWithConnectionStateSetter to ISharedTree?

Reply:
I removed `ConnectionSetter` and hence `SharedTreeWithConnectionStateSetter`which was just an extension of `ISharedTree` with `ConnectionSetter`

Reply:
I had not seen you comment on `ConnectionSetter` when I wrote this. I will comment there.